testing_utils/check_covidcast_port.py

```diff
@@ -0,0 +1,169 @@
+"""
+script to check converting covidcast api calls with Epidata.covidcast Epidata.covidcast_meta
+"""
```

Comment:
Do these tests pass locally for you? How long do they take to run?

Reply:
I'm trying to be through since sircomplainalot does take into account for all the signals, it's been some amount of minutes (mostly on the covidcast because of the for loop)

I'm working on grabbing like a few days from their respective min time

Reply:
the covidcast one takes about takes about an hour to run and epidata take about 30 minutes, so toghet about 1.5 hours